pallets/liquidity-pools-gateway/routers/src/routers/axelar_evm.rs

```diff
@@ -74,8 +74,8 @@ where
 }
 
 /// Encodes the provided message into the format required for submitting it
-/// to the Axelar contract which in turn submits it to the LiquidityPools
-/// contract.
+/// to the Axelar contract which in turn calls the LiquidityPools
+/// contract with the serialized LP message a payload.
```

Comment:
I think this is enough 👀 

```suggestion
/// contract with the serialized LP message.
```

Reply:
Ah, that should be `with the serialized LP message as payload` ...